astropy/analytic_functions/tests/test_blackbody.py

```diff
@@ -105,3 +106,14 @@ def test_blackbody_exceptions_and_warnings():
         blackbody_nu(-1. * u.AA, 5000)
     assert len(w) == 1
     assert 'invalid' in w[0].message.args[0]
+
+
+def test_blackbody_array_temperature():
+
+    # Regression test to make sure that the temperature can be an array
+
+    flux = blackbody_nu(1.2 * u.mm, [100, 200, 300])
```

Comment:
I think a better test would be an array of wavelengths *and* an array of temperatures. It is uncommon for people to only evaluate blackbody model at a single wavelength value.